plugins/catalog/src/alpha/pages.tsx

```diff
@@ -73,6 +73,9 @@ export const catalogEntityPage = PageBlueprint.makeWithOverrides({
       EntityContentBlueprint.dataRefs.filterExpression.optional(),
       EntityContentBlueprint.dataRefs.group.optional(),
     ]),
+    extraContextMenuItems: createExtensionInput([
+      coreExtensionData.reactElement,
```

Comment:
Would it make sense for this type to instead have like a title and an optional icon? And then perhaps a callback onclick function so to speak.